core.js

```diff
@@ -1162,6 +1162,13 @@ async function _fromTokenizer(tokenizer) {
 		}
 	}
 
+	if (check([0x4E, 0x49, 0x54, 0x46, 0x30])) {
```

Comment:
This is in the incorrect place. It should be grouped with the other 5-byte signatures.